### PR TITLE
reduce time unavailable

### PR DESCRIPTION
**Pull request by Idrinth**
add force-create check

### CODE REVIEW

File: bin/force-create.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+require_once dirname(__DIR__) . '/vendor/autoload.php';
+
+Dotenv::createImmutable(dirname(__DIR__))->load();
+
+$database = new PDO('mysql:dbname=' . $_ENV['DB_DATABASE'] . ';host=' . $_ENV['DB_HOST'], $_ENV['DB_USER'], $_ENV['DB_PASSWORD']);
+$needsRefresh = $database->query('SELECT 1 FROM force_refresh WHERE server=' . gethostname())->fetchColumn();
+if ($needsRefresh) {
+    $database->exec('DELETE FROM force_refresh WHERE server=' . gethostname());
+    (new VirtualHostGenerator(
+        new PDO('mysql:dbname=' . $_ENV['DB_DATABASE'] . ';host=' . $_ENV['DB_HOST'], $_ENV['DB_USER'], $_ENV['DB_PASSWORD']),
```
Comment (JEM94) [inline bin/force-create.php]: why isn't $database reused?

File: src/VirtualHostGenerator.php [hunk 2]
```diff
@@ -127,7 +127,10 @@ private function buildServerHost(string $hostname, string $admin, array &$virtua
     }
     public function create()
     {
-        exec("service apache2 stop");
+        exec("a2ensite https-only");
```
Comment (JEM94) [inline src/VirtualHostGenerator.php]: why is https-only enabled and disabled in the end? http-only gets disabled and enabled

**Review by JEM94 — CHANGES_REQUESTED** [review]

File: bin/force-create.php (hunk 1 above)
Comment (Idrinth) [reply bin/force-create.php]: fixed

File: src/VirtualHostGenerator.php (hunk 2 above)
Comment (Idrinth) [reply src/VirtualHostGenerator.php]: fixed, forgot to change this case

**Review by JEM94 — APPROVED** [review]

**Pull request merged**